Dockerfile

```
@@ -130,6 +130,7 @@ USER root
 COPY --from=back-end-builder /kargo/bin/ /usr/local/bin/
 COPY --from=tools /tools/ /usr/local/bin/
 
-USER 1000:0
+RUN adduser -D kargo -u 1000
+USER kargo:0
```

```suggestion
RUN adduser -D -H -u 1000 kargo
USER 1000:0
```

Addition of `-H` prevents creation of a home directory for this user.

I prefer that we keep `USER 1000:0` as it was. `1000` is the conventional/common non-root user ID for containers in Kubernetes. I think keeping it as `USER 1000:0` makes it more instantly recognizable that we're following that.

This `RUN adduser -D -H -u 1000 kargo` also needs to be added on line 97 or else the rest of these changes aren't going to work in our local development environments.